pkgs/games/legendary-gl/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, fetchFromGitHub
+, python38Packages
+}:
+
+with python38Packages;
+
+buildPythonApplication rec {
+  pname = "legendary-gl";  # Name in pypi
+  version = "0.0.14";
+
+  src = fetchFromGitHub {
+    owner = "derrod";
+    repo = "legendary";
+    rev = version;
+    sha256 = "05r88qi8mmbj07wxcpb3fhbl40qscbq1aqb0mnj9bpmi9gf5zll5";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  meta = with lib; {
+    description = "A free and open-source Epic Games Launcher alternative";
+    homepage = https://github.com/derrod/legendary;
```

Use quotes for URLs (RFC45)
```suggestion
    homepage = "https://github.com/derrod/legendary";
```

---

Thanks :)